### PR TITLE
1st commit

### PR DESCRIPTION
**Pull request by Kukkadapu-Ruchitha**
suggestion

### CODE REVIEW

File: ReverseArrayAndCountVowels.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.lang.String;
+
+public class ReverseArrayAndCountVowels {
+    public static void main(String[] args) {
+        System.out.println("Enter string");
+        Scanner sc = new Scanner(System.in);
+        String string = sc.nextLine();
+        reverseArrayAndNoOfVowels(string);
```
Comment (VBhaskarachary) [inline ReverseArrayAndCountVowels.java]: Please use proper naming convention.

File: ReverseArrayAndCountVowels.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    public static void main(String[] args) {
+        System.out.println("Enter string");
+        Scanner sc = new Scanner(System.in);
+        String string = sc.nextLine();
+        reverseArrayAndNoOfVowels(string);
+
+    }
+
+    private static void reverseArrayAndNoOfVowels(String string) {
+        int count = 0;
```
Comment (VBhaskarachary) [inline ReverseArrayAndCountVowels.java]: Donot use static member

File: ReverseArrayAndCountVowels.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        if (string.isEmpty()) {
+            System.out.println("Invalid Input");
+        } else {
+
+            String string2 = string.toLowerCase();
```
Comment (VBhaskarachary) [inline ReverseArrayAndCountVowels.java]: Pls proper naming in whole assignment

File: Selection.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter your choice :\n 1. Black and White TV. \n 2. Color TV");
+        String choice = sc.nextLine();
+
+        if(choice.equals("1.Black and White TV")){
+            Selection.superclass A = new Selection.superclass();
+            A.print();
+        }
```
Comment (VBhaskarachary) [inline Selection.java]: use dynamic binding

File: ReverseArrayAndCountVowels.java (hunk 1 above)
Comment (Kukkadapu-Ruchitha) [reply ReverseArrayAndCountVowels.java]: ok I modified

File: ReverseArrayAndCountVowels.java (hunk 2 above)
Comment (Kukkadapu-Ruchitha) [reply ReverseArrayAndCountVowels.java]: I modified

File: ReverseArrayAndCountVowels.java (hunk 3 above)
Comment (Kukkadapu-Ruchitha) [reply ReverseArrayAndCountVowels.java]: fixed it

File: Selection.java (hunk 4 above)
Comment (Kukkadapu-Ruchitha) [reply Selection.java]: fixed

**Review by VBhaskarachary — APPROVED** [review]
Changes looks good to me

**Pull request merged**